pmlc/conversion/gpu/enums.td

```diff
@@ -0,0 +1,21 @@
+#ifndef __PML_VULKAN_ENUMS__
+#define __PML_VULKAN_ENUMS__
+
+#ifndef OP_BASE
+include "mlir/IR/OpBase.td"
+#endif
+
+def BUFFER_COPY_MODE : BitEnumAttr<
```

Comment:
Use `CapitalCasing` instead of all caps.

Reply:
Fixed